samples/connectedhomeip/common/src/NFCWidget.h

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (c) 2020 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+ */
+
+#pragma once
```

Comment:
Doesn't Zephyr use ifdefs normally? We can leave it and wait for some comments.

Reply:
Yes, although I'm not sure the rule is written down anywhere. Should I change it?

Reply:
Let's wait for community comments.